core/CHANGELOG.md

```diff
@@ -36,13 +36,19 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v0.11.0](https://github.com/cosmos/cosmos-sdk/releases/tag/core%2Fv0.11.0)
+
+:::note
+This release contains breaking changes and should not be used with 0.50.x or earlier versions of the Cosmos SDK.
+:::
+
 * [#17468](https://github.com/cosmos/cosmos-sdk/pull/17468) Add `appmodule.HasPreBlocker` interface.
```

Comment:
We need pre blocker for v0.50.
Can we update to tag 11 with that and v12 with the comet changes?